tests/app/test_party_service.py

```diff
@@ -109,7 +109,7 @@ def test_get_user_details_for_bres_user(self, mock_request):
         mock_request.get(user_data_url, status_code=200, reason="OK", text=str(expected_result))
 
         with self.app.app_context():
-            result_data = sut.get_user_details(constants.BRES_USER)
+            result_data = sut.get_user_details(constants.NON_SPECIFIC_INTERNAL_USER)
```

Comment:
Looking at the `get_user_details` method it looks like the party service was never called for the `BRES_USER` or `NON_SPECIFIC_INTERNAL_USER`. If so, now would be a good time to remove the functionality from the secure message party service code and then this test wouldn't be needed.